### ISSUE TITLE
People not getting on elevator

### ISSUE THREAD

**Issue by aonsager** [magwo/elevatorsaga#18]
It seems like there are certain cases where people will not get on elevators. 

This is the simplest way to reproduce it that I've found:

```javascript
{
    init: function(elevators, floors) {
        var elevator = elevators[0]; // Let's use the first elevator
        elevator.on("idle", function() {
            // The elevator is idle, so let's go to all the floors (or did we forget one?)

        });
    },
    update: function(dt, elevators, floors) {
        // We normally don't need to do anything here
    }
}
```

Adding `elevator.goToFloor(0);` to the idle event will make them get on.

**Comment by magwo** [comment]
I noticed it too. I think it's because the users (passengers) will only enter an elevator upon the "entrance_available" event emitted by elevators. So if the elevator is just standing still on a floor, it doesn't emit any events, and the user doesn't get on. Not sure how to best solve it.

**Comment by frevds** [comment]
do NOT announce the direction by using the indicator functions - helped me, strange though.

**Comment by lakario** [comment]
My solution calls goToFloor(), regardless of whether or not the elevator is currently on that floor (assuming that it's not already going there). This approach seems to prevent any idle elevators.

**Comment by g-p-g** [comment]
I didn't notice this yesterday, now I do. Maybe the elevator is not opening its door and that's why people don't get in? :)

**Comment by qiuzuhui** [comment]
+1 
and elevator indicator is wrong ? 
elevator.js: line:220
        elevator.trigger("indicatorstate_change", {up: elevator.goingDownIndicator, down: elevator.goingUpIndicator})

**Comment by roryokane** [comment]
@qiuzuhui That separate bug about the elevator indicators has been fixed in #32.

**Comment by magwo** [comment]
https://github.com/magwo/elevatorsaga/commit/6ce62a1481d8df5cfd73f45851cde46ff68d41e1
